src/client/java/net/neoforged/neoforge/client/event/ScreenEvent.java

```diff
@@ -209,6 +209,24 @@ public Pre(Screen screen, GuiGraphics guiGraphics, int mouseX, int mouseY, float
             }
         }
 
+        /**
+         * Fired <b>after</b> the screen's renderBackground and <b>before</b> its render method.
+         *
+         * <p>This can be used for rendering elements that must be below tooltips and the dragged stack,
+         * such as slot or item stack specific overlays.</p>
+         *
+         * <p>This event is not {@linkplain ICancellableEvent cancellable}, and does not {@linkplain HasResult have a result}.</p>
```

Comment:
```suggestion
         * <p>This event is not {@linkplain ICancellableEvent cancellable}.</p>
```
The generic event result system doesn't exist anymore.